packages/office-ui-fabric-react/package.json

```diff
@@ -71,6 +71,7 @@
     "sinon": "^4.1.3"
   },
   "dependencies": {
+    "@fluentui/react-focus": "^0.0.2",
```

Comment:
Should we bump this to v7 before adding it as a dependency of fabric? Once it's re-exported from Fabric, we lose the option of doing any breaking changes outside of a major version, and therefore it's no longer truly a v0.

Reply:
Done!